pbt-libraries/americium/reports/deletion.md

```diff
@@ -0,0 +1,22 @@
+# Report for Americium Shrinking on "deletion"
+
+This report was generated with Americium 1.4.2
+
+## Normalization
+
+Americium currently uses a fixed internal randomisation seed, so always yields the same shrinkage:
```

Comment:
Just shrinking with a given random seed seems to partially defy the purpose of the shrinking challenge. I guess you should try to move towards non-fixed seeds in the long run.